AppController/lib/app_manager_client.rb

```diff
@@ -23,7 +23,7 @@ class AppManagerClient
   attr_reader :conn, :ip
 
   # The port that the AppManager binds to, by default.
-  SERVER_PORT = 49934
+  SERVER_PORT = 17445
```

Comment:
Will the existing monit config file (suffix: -49934.cfg) be replaced upon down/up.?